src/applications/vaos/containers/AppointmentsPage.jsx

```diff
@@ -60,33 +65,41 @@ export class AppointmentsPage extends Component {
       showScheduleButton,
       showCommunityCare,
       hasExpressCareAccess,
+      expressCare,
       showDirectScheduling,
       isCernerOnlyPatient,
       showPastAppointments,
     } = this.props;
+    const { fetchWindowsStatus, allowRequests } = expressCare;
 
     return (
       <div className="vads-l-grid-container vads-u-padding-x--2p5 large-screen:vads-u-padding-x--0 vads-u-padding-bottom--2p5">
         <Breadcrumbs />
         <div className="vads-l-row">
           <div className="vads-l-col--12 medium-screen:vads-l-col--8 vads-u-margin-bottom--2">
             <h1 className="vads-u-flex--1">{pageTitle}</h1>
-            {showScheduleButton && (
-              <ScheduleNewAppointment
-                isCernerOnlyPatient={isCernerOnlyPatient}
-                showCommunityCare={showCommunityCare}
-                showDirectScheduling={showDirectScheduling}
-                startNewAppointmentFlow={this.startNewAppointmentFlow}
-              />
-            )}
-            {hasExpressCareAccess && (
+            {fetchWindowsStatus === FETCH_STATUS.loading ? (
+              <div className="vads-u-margin-y--8">
+                <LoadingIndicator />
+              </div>
+            ) : (
               <>
-                <RequestExpressCare />
-                <h2 className="vads-u-font-size--h3 vads-u-margin-y--3">
-                  View your upcoming, past, and Express Care appointments
-                </h2>
+                {showScheduleButton && (
```

Comment:
What do you think about letting the new appointment button show up while the check is running? That's what I was expecting in the tab changes PR I have.

Reply:
Sure, sounds good.